### PR TITLE
Migrate project to the AceEditorBundle\AceEditorBundle namespace (fixes #74)

### LINKED ISSUE #74: Migrate project to a different namespace.
I chose this namespace a long time ago but it feels just a bit too complicated for almost anyone who is not speaking Polish 😅
Would be much better to either move it under NorbertTech namespace or something completely different like AceEditorPhp maybe (I'm open for any other propositions). 

Version 5.0.0 might be a good opportunity for that, since it would also require changing the package name and updating packagist.org

### PR DESCRIPTION
**Pull request by nocive**
Fixes #74

### CODE REVIEW

**Comment by nocive** [comment]
cc @norberttech

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Default configuration:
 ```
 # app/config/config.yml
 
-norzechowicz_ace_editor:
+norberttech_ace_editor:
```
Comment (nocive) [inline README.md]: `norberttech_symfony_ace_editor` would be more correct but feels too long?

**Comment by norberttech** [comment]
Hey! Sorry I was out for a holidays break 🎅
In general `norberttech_ace_editor` vs `norberttech_symfony_ace_editor` shows that using my username in the namespace might not be the best approach. Maybe just AceEditorBundle and then `ace_editor` ?

**Comment by nocive** [comment]
@norberttech updated. I kept the internal parameters with the `norbertech` prefix though, as to avoid potential naming collisions with other implementations in the wild.

**Comment by norberttech** [comment]
> @norberttech updated. I kept the internal parameters with the `norbertech` prefix though, as to avoid potential naming collisions with other implementations in the wild.

Are there even any other implementations? 😅 I think I would keep it simple, `ace_editor`, not sure what would be the use case to have two bundles doing the same thing installed.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ bundle yourself:
 
 return [
     // ...
-    Norzechowicz\AceEditorBundle\NorzechowiczAceEditorBundle::class => ['all' => true],
+    NorbertTech\AceEditorBundle\NorbertTechAceEditorBundle::class => ['all' => true],
```
Comment (norberttech) [inline README.md]: I would move it under `AceEditorBundle\AceEditorBundle::class` - this way it becomes completely agnostic and unrelated to my name.

Comment (nocive) [reply README.md]: fair enough, will do.

**Comment by nocive** [comment]
> Are there even any other implementations? 😅 I think I would keep it simple, `ace_editor`, not sure what would be the use case to have two bundles doing the same thing installed.

You have no idea what people are capable of :sweat_smile:

**Comment by norberttech** [comment]
> > Are there even any other implementations? 😅 I think I would keep it simple, `ace_editor`, not sure what would be the use case to have two bundles doing the same thing installed.
> 
> You have no idea what people are capable of 😅

Hahaha, yeah I have seen things that can't be unseen, but let's not go there, it's an edge case which I don't think is worth our time 😁

**Comment by nocive** [comment]
updated

**Review by norberttech — APPROVED** [review]

**Pull request merged**